File: 백준/Bronze/5585. 거스름돈/거스름돈.py
num = int(input()) #내가 지불할 금액 
change = 1000 - num #1000 - 내가 지불할 금액 빼기 

coins = [500, 100, 50, 10, 5, 1]
count = 0 #거스름돈 개수 확인

for coin in coins:
    count += change // coin 
    change %= coin 
    
print(count)

File: 백준/Bronze/5585. 거스름돈/거스름돈.js
let fs = require('fs')
let input = fs.readFileSync('/dev/stdin').toString().split('\n');

let money = parseInt(input[0])
let change = 1000 - money 
let count = 0
let coins = [500, 100, 50, 10, 5, 1]

for(let coin of coins){
    count += Math.floor(change/coin);
    change %= coin
}
console.log(count)